src/json.test.ts

```diff
@@ -39,6 +50,143 @@ jest.mock('superstruct', () => ({
   assert: jest.fn(),
 }));
 
+describe('object', () => {
+  it('validates an object', () => {
+    expect(
+      is(
+        {
+          foo: 'bar',
+        },
+        object({
+          foo: string(),
+        }),
+      ),
+    ).toBe(true);
+
+    expect(
+      is(
+        {
+          foo: 123,
+        },
+        object({
+          foo: string(),
+        }),
+      ),
+    ).toBe(false);
+  });
+
+  it('validates an object with optional values', () => {
+    expect(
+      is(
+        {
+          foo: 'bar',
+        },
+        object({
+          foo: exactOptional(string()),
+        }),
+      ),
+    ).toBe(true);
+
+    expect(
+      is(
+        {},
+        object({
+          foo: exactOptional(string()),
+        }),
+      ),
+    ).toBe(true);
+
+    expect(
+      is(
+        {
+          foo: 123,
+        },
+        object({
+          foo: string(),
```

Comment:
Should this example be in this test? It doesn't seem to be related to optional / exact optional.

Reply:
I added it as an integration test kind of thing, to make sure that the `object` struct still works with regular structs. I can remove it if you prefer.

Reply:
Okay. That's fine, but what do you think about putting it in its own test?